.github/workflows/deploy-gcp-tests.yml

```diff
@@ -901,9 +908,9 @@ jobs:
               --source-disk-zone=${{ vars.GCP_ZONE }} \
```

Comment:
To test these changes, I need to run CI on this branch, and force it to save a disk with the new label:
https://github.com/ZcashFoundation/zebra/actions/runs/6179715964

<img width="321" alt="Screenshot 2023-09-14 at 11 18 37" src="https://github.com/ZcashFoundation/zebra/assets/8951843/40108e88-5be9-46a9-a6f1-befd9ca3671b">

#### Testing

The disks created by that action run must have an `updated-from-disk` label.
The "Get original cached state height from google cloud" step should log the disk name and height correctly.